### PR TITLE
More documents for the top level directory.

### PR DESCRIPTION
**Pull request by afs**
See also JENA-1234.

This PR include BUILD.md and CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Documentation is kept and published with Apache's svnpubsub system:
+
+   https://svn.apache.org/repos/asf/jena/site/trunk
+
+## Workflow
+
+### JIRA
+
+The project uses a JIRA instance to track work.  Please create a JITA so
```
Comment (kinow) [inline CONTRIBUTING.md]: s/JITA/JIRA

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+that we can track a contribution.
+
+   https://issues.apache.org/jira/browse/JENA
+
+### Github
+
+It is useful to create a JIRA then use teh JIRA number (e.g. JENA-9999)
```
Comment (kinow) [inline CONTRIBUTING.md]: s/teh/the

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+in the Pull Request title. This activates the automated mirroring of
+discussions onto the project developers mailing list.
+
+To make a contribution:
+
+* On github, fork http://github.com/apache/jena into you github account.
+* Create a branch in your fork for hthe contribution.
```
Comment (kinow) [inline CONTRIBUTING.md]: s/hthe/the

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+* Make your changes. Include the Apache source header at the top of each file.
+* Generate a pull request via github. Further changes to your branch will automatically
+  show up in the pull request
+
+The project development mailing list is automatically notified of new pull
+requests and JIRA ia also automatically updated if the JIRA id is in the pull request
```
Comment (kinow) [inline CONTRIBUTING.md]: s/ia/is

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+for generics, and don't declared checked exceptions that are not used.
+Use `@SuppressWarnings("unused")` as necessary.
+
+Please don't mix reformatting and functional changes; it makes it harder
+to review.
+
+### Legal
+
+All contributions are understood to be made as contributions to the
+Apache Software Foundation by the person makign the pull request. These
```
Comment (kinow) [inline CONTRIBUTING.md]: s/makign/making

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+are then incorporated into the project codebase and licnesed under the
```
Comment (kinow) [inline CONTRIBUTING.md]: s/licnesed/licensed

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+Apache License 2.0.
+
+You, as an individual, must be entitled to make the contribution to the
+project. If the contrbution is part of your employment, please arrange
```
Comment (kinow) [inline CONTRIBUTING.md]: s/contrbution/contribution

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+this before making the contribution.
+
+For a large contribution, the project may ask for a specific Software
+Grant from the contributor.
+
+If in doubt, or you have any questions, ask on the dev@jena.apache.org
```
Comment (kinow) [inline CONTRIBUTING.md]: "If in doubt, or if you have any questions" perhaps?

**Comment by rvesse** [comment]
Looks good to me

File: CONTRIBUTING.md (hunk 8 above)
Comment (afs) [inline CONTRIBUTING.md]: All changes done - thanks.

**Pull request merged**